extension/ezoe/design/standard/templates/ezoe/customattributes/link.tpl

```diff
@@ -90,6 +90,7 @@ eZOEPopupUtils.settings.onInitDoneArray.push( function( editorElement )
         ezoeLinkAttribute.timeOut = setTimeout( eZOEPopupUtils.BIND( ezoeLinkAttribute.ajaxCheck, this, url[0] + '_' + id, lid ), 320 );
         return true;
     });
+    inp.keyup();
```

Comment:
@L0rdJ I think you accidentally merge in unrelated changed here, or?

Reply:
@andrerom Sorry, you are correct. It was unrelated change. This change was related to https://github.com/ezsystems/ezpublish-legacy/pull/1291.

Reply:
Ok, merged that first so it got resolved when merging this, thanks @L0rdJ !